### ISSUE TITLE
Authorized link issue

### ISSUE THREAD

**Issue by magateamdev** [stranoweb/stranoweb-ajax-login#10]
Hi,
I have a problems with Google 'Authorized JavaScript origins' and 'Authorized redirect URI' fields and Facebook 'Site URL' field. The fields are empty.
For Test, I installed the plugin on new and clear wordpress with same result.

![image](https://user-images.githubusercontent.com/32236437/73169022-cd1a9480-4103-11ea-81fb-56b4103a00ba.png)
![image](https://user-images.githubusercontent.com/32236437/73169089-f63b2500-4103-11ea-89e2-e64a1d77a26d.png)

**Comment by stranoweb** [comment]
Hi, thanks for reporting the issue.
Will be fixed and released shortly.
That link is the home_url()

**Comment by magateamdev** [comment]
thanks
i’ll try

**Comment by magateamdev** [comment]
Hi
I steel have issue with Google Authorization.  After Google authentication, user get this error:
"Error : Failed to get user information".
It's happen in several sites with this plugin..
i get another social login plugin and it is working fine with same "Client ID"

I think that a problem is "Authorized redirect URI" 

need your help to solve this issue

thanks

**Comment by stranoweb** [comment]
Hi, sorry for the long await, we are about to release the biggest update in the next days with a lot of new features and that fix.
Here is how the URL must be, without the last /. Change our URL with yours.
![4BAAB09F-6C53-4F27-8495-944E03BDA426](https://user-images.githubusercontent.com/35683062/74478940-a2c02980-4eae-11ea-8727-e107ac00b920.png)

**Comment by magateamdev** [comment]
it is exactly what i did
but i still have this problem: "Error : Failed to get user information".

**Comment by torokzoli** [comment]
I have the same issue

**Comment by stranoweb** [comment]
We have just released the new update 1.5.0.
Please go to our website https://www.ajaxlogin.com on your download page, download it and overwrite the old files with the new ones.
The new version requires license key you will find on License Keys page on your account page.

Let me know if you still have the issue.

**Comment by magateamdev** [comment]
Hi
Sorry, but the issue still exists.
I also installed a clean new WORDPRESS without any plugins and themes and I tried the plugin with the same result.

![image](https://user-images.githubusercontent.com/32236437/75140235-27a11380-56f7-11ea-915c-07743ab77b68.png)

I installed some free plugins from wordpress.org and used the same OAuth 2.0 Client ID, and everything worked fine for me.

If you want I can give you access to my test site to check the issue.
